README.md

```diff
@@ -57,4 +57,6 @@ correct version of `pycmake` into your *libecl* code:
     git submodule update --init pycmake
 
 
-
+### Installing the Python code ###
+    * pip install .
```

Comment:
Isn't this slightly misleading? The `pip install` suggestion/alternative is for people who are not interested in downloading the code at all - and just want the simple & fast pip based approach?

Reply:
done